system/include/emscripten/fetch.h

```diff
@@ -93,8 +93,10 @@ typedef struct emscripten_fetch_attr_t
 
 	// If non-zero, specifies a pointer to the data that is to be passed as the body (payload) of the request
 	// that is being performed. Leave as zero if no request body needs to be sent.
-	// The memory pointed to by this field is provided by the user, and needs to be valid only until the call to
-	// emscripten_fetch() returns.
+    // The memory pointed to by this field is provided by the user, and needs to be valid throughout the
+	// duration of the fetch operation. If passing a non-zero pointer into this field, make sure to implement
+	// the onsuccess and onerror handlers to be notified when the fetch finishes to know when this memory
```

Comment:
Could this line stress the need to implement both the handlers, e.g. with `make sure to implement *both* the onsuccess and onerror handlers`.